src/views/homepage/server.ts

```diff
@@ -0,0 +1,50 @@
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+import { getInlineContentMaps } from 'lib/tutorials/get-inline-content-maps'
+import {
+  FeaturedLearnCard,
+  FeaturedLearnContent,
+} from 'views/product-downloads-view/types'
+import { formatCollectionCard } from 'components/collection-card/helpers'
+import { formatTutorialCard } from 'components/tutorial-card/helpers'
+import {
+  GenerateStaticPropsOptions,
+  GenerateStaticPropsOptionsResult,
+} from './types'
+
+const generateStaticProps = async ({
+  featuredLearnContent = [],
+}: GenerateStaticPropsOptions): Promise<GenerateStaticPropsOptionsResult> => {
+  // Gather tutorials and collections based on slugs used
+  const { inlineCollections, inlineTutorials } = await getInlineContentMaps(
+    featuredLearnContent
+  )
+
+  // Transform featured tutorial and collection entries into card data
+  const featuredLearnCards: FeaturedLearnCard[] = featuredLearnContent.map(
```

Comment:
If we keep this here we could note the duplication of this code in the install view, just as a 🔖 to be aware of the spread.